### ISSUE TITLE
nginx issue

### ISSUE THREAD

**Issue by devads** [61FINTECH/deploy-strapi-on-aws#2]
Hi,

I tried to add nginx.conf to root folder on project but always got the default page of nginx, I tried to change root directory in /etc/nginx/sites-available/default but front is accessible but not admin :-(

Thanks,

**Comment by kenberkeley** [comment]
strapi-cms/nginx.conf is used to replace /etc/nginx/nginx.conf, which needs sudo privilege.

**Comment by devads** [comment]
Sorry, ok I replaced /etc/nginx/nginx.conf, it's working now !

**Comment by devads** [comment]
I think it's a nginx conf but now I got an error on each page :
502 Bad Gateway
I use same nginx.conf as you, SSL certificate are OK

**Comment by kenberkeley** [comment]
Must be a server down, restart it!

**Comment by kenberkeley** [comment]
I mean Strapi, not Nginx

**Comment by devads** [comment]
Strapi servers are OK (prod, dev and staging) and I already tried to restart them
`pm2 list`
status online
Thanks,

**Comment by kenberkeley** [comment]
Open the ports of the firewall, and visit them directly

**Comment by devads** [comment]
All ports of the firewall were already opened (for testing purpose)

**Comment by kenberkeley** [comment]
Can you access http://[ServerIP]:1337 1338 1339?

**Comment by devads** [comment]
not really indeed, but strapi service is OK

**Comment by kenberkeley** [comment]
Well...

**Comment by kenberkeley** [comment]
![image](https://user-images.githubusercontent.com/13287948/55791458-dfd6c500-5b01-11e9-9983-73e6a3d80434.png)

What have you done after this...

**Comment by devads** [comment]
Without ssl it was working, with SSL I got this 502 error.
Anyway, I switched to d2c they got strapi stack hub :-)

Thanks anyway !

**Comment by steveafrost** [comment]
> Without ssl it was working, with SSL I got this 502 error.
> Anyway, I switched to d2c they got strapi stack hub :-)
> 
> Thanks anyway !

I'm running across the same issue. With SSL I get 502 Bad Gateway. I'm unable to access the Strapi process by IP & port directly to answer @kenberkeley's last question.

**Comment by kenberkeley** [comment]
Well, maybe there are updates from Strapi team that breaks everything.

**Comment by steveafrost** [comment]
Yeah, lots of updates lately w/ beta coming out. I was eventually able to reach via IP & port (own mistake of not building for production) but the nginx config is still throwing 503 gateway errors. If I figure it out, I'll post back.

Thanks for the tutorial/walkthrough! Super helpful.

**Comment by kenberkeley** [comment]
@steveafrost PR is welcome
